vcpkg.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": [
+    "curl",
+    "freetype",
+    "gettext",
+    "gmp",
+    "jsoncpp",
+    "libjpeg-turbo",
+    "libogg",
+    "libvorbis",
+    "luajit",
+    "openal-soft",
+    "opengl-registry",
```

Comment:
I belive opengl-registry is not needed anymore

Reply:
Sure, I'll try rebuilding without this one